api/xds/gloo-discovery-service.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package xds.solo.io;
```

Comment:
solo.io.xds?

Reply:
just curious, why don't we follow the convention we use elsewhere for package names, which goes from most specific -> least specific, for example `xds.solo.io`?